### DISCUSSION TITLE
MyST Markdown: Collapsible section/block

### DISCUSSION THREAD

**Discussion by CodeCox** [executablebooks/meta#116] [Ideas] [answered]
I can write this `html` in an .md file
```html
<details>
<summary>Click me</summary>
TADA!!! This is some *hidden* text.  I'm usually lengthy but this is a silly example. 
And I also want to be ** MyST markdown**. 
</details>
```
which results in this 
<details>
<summary>Click me</summary>
TADA!!! This is some *hidden* text.  I'm usually lengthy but this is a silly example. 
And I also want to be ** MyST markdown**. 
</details>

**Question:** Can I do this (ie. *expand/collapse*) in MyST markdown without resorting to `html`? (Perhaps by using an existing or a custom directive?)

I noticed that an `ipynb` cell can be tagged as **scrollable** which is also a way to deal with optimising premium space. But my use case is different - I want to hide a block of text until the reader actively solicits further info. For example, a Financial Report may contain both *business* and *technical* parts, and the tech bits could be collapsed so that both types of audiences are catered to. For Intsance, an Executive might not expand the tech sections, or may only do so if time permits. 

As Github users - this is of great value when creating an issue - improves readabilty (Tracebacks I'm looking at you! )

**Comment by chrisjsewell** [comment, marked as answer]
> Question: Can I do this (ie. expand/collapse) in MyST markdown without resorting to html? (Perhaps by using an existing or a custom directive?)

You can indeed 😄 https://sphinx-panels.readthedocs.io/en/latest/#dropdown-usage

**Reply by chrisjsewell** [reply]
I'm going to add an example of this in the jupyter-books documentation now actually (and make it pre-installed). Thanks for reminding me!

**Reply by CodeCox** [reply]
Great - I'll circle back to look at the jb doc later!

**Comment by CodeCox** [comment]
I'm test-driving this feature now in **v0.8.0**! 

`Sphinx-Panel Integration` is much more than I expected!
